phpunit.xml.dist

```diff
@@ -32,14 +51,5 @@
             <directory>./tests/Context/Unit</directory>
         </testsuite>
     </testsuites>
-    <filter>
-        <whitelist>
-            <directory>contrib</directory>
-            <directory>sdk</directory>
-        </whitelist>
-    </filter>
 
-    <logging>
```

Comment:
For some reasons PhpUnit 9 generated an error that said that "<log>" element is not expected. The docs says it must be okay. I googled it for a while but havenn't found anything on it. I dont' feel like going deep into this, instead I just moved logging config to the CLI option. Looks like its fine now

Reply:
Sounds good